modules/oe_webtools_media/src/Plugin/media/Source/Webtools.php

```diff
@@ -125,6 +125,7 @@ public function getWidgetTypes() {
       'chart' => [
         'name' => $this->t('Chart'),
         'service' => 'charts',
+        'services' => ['chart', 'racing'],
```

Comment:
If `service` is deprecated then need to provide `services` parameter for each widget type as was done in the first commits.

Reply:
yes, all of them need to get `services`